src/views/pages/home/index.pug

```diff
@@ -11,4 +12,6 @@ block content
         h2.heading.foobar Hello world 
         div#potato.chips
             p Hot Potatoes
-            +test()
\ No newline at end of file
+            +test()
```

Comment:
I think we can get rid of this test content - do you want to do it here?

Reply:
Removed test and elements